securedrop/source.py

```diff
@@ -313,14 +313,14 @@ def submit():
 
     else:
         if msg and not fh:
-            things = 'message'
+            html_contents = 'Thanks! We received your message.'
```

Comment:
gettext('Thanks! We received your message.')

and the same for the strings below. Which makes me realize these were not i18n...

Reply:
Fixed